### PR TITLE
Workaround and extra logging to investigate bug in log rotation

### PR DESCRIPTION
**Pull request by mambelli**
Added workaround and extra logging to investigate a bug in log rotation
Added also some compatibility adaptors to run unit tests on a Mac

This improves issue #362

### CODE REVIEW

File: factory/glideFactoryEntryGroup.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
 from glideinwms.factory import glideFactoryLib as gfl
 from glideinwms.factory import glideFactoryPidLib
 from glideinwms.lib import classadSupport, cleanupSupport, logSupport
-from glideinwms.lib.fork import fetch_fork_result_list, ForkManager
+from glideinwms.lib.fork import fetch_fork_result_list, ForkManager, print_child_processes
```
Comment (mmascher) [inline factory/glideFactoryEntryGroup.py:32]: Isn't this a syntax error? I mean the comma after ForkManager?

**Review by mmascher — COMMENTED** [review]
Everything else is good, just check the syntax error of the earlier comment.

File: factory/glideFactoryEntryGroup.py (hunk 1 above)
Comment (mambelli) [reply factory/glideFactoryEntryGroup.py:32]: I don't think. And pylint and the unit test are OK. The comma should be after all items after the first one: https://docs.python.org/3/reference/simple_stmts.html#the-import-statement

Comment (mmascher) [reply factory/glideFactoryEntryGroup.py:32]: For some reason I saw `ForkManager` written in red and my brain registered it as the`import keyword`. all good.

**Review by mmascher — APPROVED** [review]

**Pull request merged**